CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ option(IREE_ENABLE_TRACING "Enables WTF tracing." OFF)
 
 option(IREE_BUILD_COMPILER "Builds the IREE compiler." ON)
 option(IREE_BUILD_TESTS "Builds IREE unit tests." ON)
+option(IREE_BUILD_DOCS "Builds IREE docs." OFF)
```

Comment:
At least so far, we've also been explicitly listing the settings in the CI script, so it's not dependent on the defaults. Not sure if that will continue to make sense:

https://github.com/google/iree/blob/0ef0256ef4c2f2bd58d7e061766887e48630b7ef/build_tools/cmake/rebuild.sh#L38

Reply:
Because of the default and the unchanged `rebuild.sh`, the `iree-doc` target wasn't build.

Just did a full rebuild (make based as I don't use the scripts normally), but passing `-DIREE_BUILD_DOCS=ON` did not result in automatic generation of docs. Instead I needed to run `make iree-doc` manually. I think this is not intended?

Reply:
Just did a full rebuild (`make` based as I don't use the scripts normally), but passing `-DIREE_BUILD_DOCS=ON` did not result in automatic generation of docs. Instead I needed to run `make iree-doc` manually. I think this is not intended?

Reply:
Good point. I added `iree-doc` to `ALL`.

Reply:
Updated `rebuild.sh`